app/models/organisation.rb

```diff
@@ -1,4 +1,11 @@
 class Organisation < ActiveRecord::Base
+  ORGANISATION_TYPES = {
+    "Call centre" => "call_centre",
+    "Custody suite" => "custody_suite",
+    "Law firm" => "law_firm",
+    "Law office" => "law_office"
+  }
```

Comment:
Would be better if this was an array and translate the labels in a view helper prehaps?